examples/iam-assumable-role-with-oidc/main.tf

```diff
@@ -50,3 +50,22 @@ module "iam_assumable_role_self_assume" {
 
   oidc_fully_qualified_subjects = ["system:serviceaccount:default:sa1", "system:serviceaccount:default:sa2"]
 }
+
+#####################################
+# IAM assumable role with audience
```

Comment:
This example doesn't match the changes being made

Reply:
Should be fixed now